### DISCUSSION TITLE
How can I setup auto-forwarding for emails from a certain address?

### DISCUSSION THREAD

**Discussion by tif-calin** [tutao/tutanota#4284] [Q&A]
This is becoming a big issue for me and would love any help with this...

**Comment by Ssieth** [comment]
This would be a really handy feature for me as well, possibly an alternative for the inbox filtering to allow for forwarding to an address rather than moving to a folder?

**Comment by DwukwiatRincewind** [comment]
Yes it's very important. Protonmail has this option... :(

**Comment by barelybearbones** [comment]
I would also like a feature like this. I'm using this for my small startup company and it would be really amazing to hate it for organization.
